web/client/components/mapcontrols/search/SearchBar.jsx

```diff
@@ -167,7 +170,7 @@ class SearchBar extends React.Component {
 
     renderAddonAfter = () => {
         const remove = <Glyphicon className="searchclear" glyph={this.props.removeIcon} onClick={this.clearSearch} key="searchbar_remove_glyphicon"/>;
-        const search = <Glyphicon className="magnifying-glass" glyph={this.props.searchIcon} key="searchbar_search_glyphicon" onClick={this.search}/>;
+        const search = <Glyphicon className={!!this.props.isSearchClickable ? "magnifying-glass clickable" : "magnifying-glass"} glyph={this.props.searchIcon} key="searchbar_search_glyphicon" onClick={this.search}/>;
```

Comment:
onclick is always set, also if isSearchClickable is false

Reply:
i see your point, i'll add the onClick method only if this isSearchClickable is true, otherwise it makes no sense to have a clickable button with a non-clickable style